### PR TITLE
Fix alter append by traditional tree parsing

### PR DESCRIPTION
**Pull request by minhaj10p**

### CODE REVIEW

File: generator/profile.go [hunk 1]
```diff
@@ -3,41 +3,107 @@ package generator
...
+		}
+		if !found {
+			continue
+		}
+		return alt, true, nil
+	}
+	return nil, false, nil
+}
+
+//EquateAlter equates alter with call
```
Comment (haroonKhan-10p) [inline generator/profile.go]:
```suggestion
// EquateAlter equates alter with call
```

**Comment by minhaj10p** [comment]
@anweiss please wait for @farhan-khan30 testing approval before you merge this. Thanks.

File: generator/mapper.go [hunk 2]
```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
 	"os"
 	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
```
Comment (anweiss) [inline generator/mapper.go]: Remove this extra line break

File: generator/reader.go [hunk 3]
```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
```
Comment (anweiss) [inline generator/reader.go]: Remove extra line break

**Comment by anweiss** [comment]
@minhaj10p @farhan-khan30 let me know if this is good to merge.

**Comment by farhan-khan30** [comment]
@anweiss LGTM. Pushed some changes, please review gain.
Commit: https://github.com/docker/oscalkit/pull/71/commits/a504cf2581e45be0896d7a75589fba4524c31658

**Pull request merged**